<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionForm</class>
 <widget class="QDialog" name="ConnectionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>875</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>配置名称</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>服务器地址</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="addr"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>服务器端口</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="port"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>密钥</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="password"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="ssl_settings_show">
     <property name="text">
      <string>显示ssl设置</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="ssl_settings">
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QFormLayout" name="ssl_settings_form1">
          <item row="0" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>verify</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="verify">
            <property name="text">
             <string>启用</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>verify_hostname</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="verify_hostname">
            <property name="text">
             <string>启用</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>prefer_server_cipher</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="prefer_server_cipher">
            <property name="text">
             <string>启用</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="ssl_settings_form2">
          <item row="0" column="0">
           <widget class="QLabel" name="qlabel">
            <property name="text">
             <string>session_ticket</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="session_ticket">
            <property name="text">
             <string>启用</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>reuse_session</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="reuse_session">
            <property name="text">
             <string>启用</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="ssl_settings_form3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>sni</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="sni"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>cert</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QWidget" name="widget_11" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="cert"/>
          </item>
          <item>
           <widget class="QToolButton" name="cert_file_open">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>key</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="key"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>key_password</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="key_password"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>cipher</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="cipher"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_32">
         <property name="text">
          <string>curves</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="curves"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>plain_http_response</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="plain_http_response"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_36">
         <property name="text">
          <string>fingerprint</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="fingerprint"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="ssl_settings_form4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>alpn</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widget_8" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QListWidget" name="alpn_list"/>
          </item>
          <item>
           <widget class="QWidget" name="widget_9" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="add_alpn_item">
               <property name="text">
                <string>添加</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="del_alpn_item">
               <property name="text">
                <string>删除</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_10" native="true"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="resetSSL">
        <property name="text">
         <string>重置ssl设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ok">
        <property name="text">
         <string>ok</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel">
        <property name="text">
         <string>cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
